src/server/controllers/match.controller.js

```diff
@@ -227,9 +241,9 @@ export function countExpectedExpertiseMatching(req, res, next) {
     .then((careerFilteredList) => {
       return new Promise((resolve, reject) => {
         let countResult = [0, 0, 0, 0, 0, 0, 0];
-        careerFilteredList.forEach((user) => {
+        careerFilteredList.full.forEach((user) => {
           user.expertise.forEach((expItem) => {
-            countResult[expItem.index]++;
+            countResult[expItem - 1]++;
```

Comment:
Similar but not exactly same. Originally, the JSON structure of expItem was like this:
```
{
  index: 0,
  content: "blah",
}
```
So we counted with `index`'s value before. But we treat and compare all of the filter data (career and expertise) as a number from this PR.

Now, you'll curious why `-1` is needed. Currently, `0` means `All` in but we don't need to return the counted valued for expertise `All`. That's why.